decisions/Barding Realm.txt

```diff
@@ -1079,8 +1079,8 @@ political_decisions = {
 			OR = {
 				population_2 = 1
 				AND = {
-					ai = yes
 					year = 2970
+					ai = yes
```

Comment:
Assuming both a human and an AI would pick the decision the moment it becomes available, the potential would fail as it checks for NOT = { has_global_flag = DAL_reforms_bard_i }, which the decision sets. So it would only be worse, if a human plays Dale and does not research the needed tech for over 14 years?

Reply:
Or rather spends more time in game between 2970 and when the tech gets researched compared to when the game started.